###
###	To upgrade mpir's yasm.
###
###	change to the mpir directory
###
###	svn del yasm
###	svn commit -m "delete old yasm"
###	svn co http://www.tortall.net/svn/yasm/trunk/yasm yasm   or  get yasm-1.2.tar.gz and untar and rename directory to yasm
###	cd yasm
###	rm -r $(find -name \\.svn)
###	patch -Np1 < ../devel/yasm.diff
###	./autogen.sh		or run autoreconf
###	./configure
###	make
###	make distclean
###	rm -r autom4te.cache
###
###	and replace any symbolic links in the config directory with real files
###
###	cd ..
###	svn add yasm
###	svn commit -m "new yasm installed version number ????"
###
###
### NOTE we can get rid of the readline part , and nail part? , and add in profiling

diff -Naur yasm2227/config/config.sub mpiryasm/config/config.sub
--- yasm2227/config/config.sub	2010-06-25 12:59:22.000000000 +0100
+++ mpiryasm/config/config.sub	2010-06-25 13:00:42.000000000 +0100
@@ -1073,7 +1073,7 @@
 	pmac | pmac-mpw)
 		basic_machine=powerpc-apple
 		;;
-	*-unknown)
+	*-unknown | *-pc | *-apple | *-w64)
 		# Make sure to match an already-canonicalized machine name.
 		;;
 	*)
diff -Naur yasm2227/configure.ac mpiryasm/configure.ac
--- yasm2227/configure.ac	2010-06-25 12:59:23.000000000 +0100
+++ mpiryasm/configure.ac	2010-06-25 13:00:42.000000000 +0100
@@ -20,6 +20,22 @@
 #
 # autoconf command-line options
 #
+AC_ARG_ENABLE(assert,,,)
+
+AC_ARG_ENABLE(alloca,,,)
+
+AC_ARG_ENABLE(cxx,,,)
+
+AC_ARG_ENABLE(fft,,,)
+
+AC_ARG_ENABLE(nails,,,)
+
+AC_ARG_ENABLE(gmpcompat,,,)
+
+AC_ARG_WITH(readline,,,)
+
+AC_ARG_ENABLE(fat,,,)
+
 AC_ARG_ENABLE(debug,
 AC_HELP_STRING([--enable-debug],
 	       [Turn on debugging and compile time warnings]),
